pyproject.toml

```diff
@@ -55,7 +55,8 @@ dependencies = [
     "koji",
     "rpkg",
     "cachetools",
-    "python-bugzilla"
+    "python-bugzilla",
+    "backoff",
```

Comment:
This needs to be added to packit.spec too.

Reply:
I checked the `packit.spec` file and I saw we have really few requires listed there (definitely not all our dependencies). I thought that most of the requires are solved by `pyproject_buildrequires`, aren't they?
Since the Copr builds in our CI were ok, I thought it sufficient.

Reply:
Correct, in line with packaging guidelines:
> Packages MUST use the automatic Python run-time dependency generator.
> Packages SHOULD use the opt-in build-dependency generator if possible.

Reply:
Indeed. It is few lines down. My bad. Ignore me.